app/src/store/initial-state/index.js

```diff
@@ -202,6 +202,7 @@ const INITIAL_STATE = {
         testThisRule: false,
         upgradeWorkspaceMenu: false,
       },
+      BillingTeamNudgeLastSeenTs: null,
```

Comment:
change to camelCase. _rename elsewhere accordingly_

```suggestion
      billingTeamNudgeLastSeenTs: null,
```